### DISCUSSION TITLE
Created New VS 2022 Task Provider for NUKE

### DISCUSSION THREAD

**Discussion by sharpninja** [nuke-build/nuke#824] [Show and tell]
I have forked [Mads's NPM Task Provider](https://github.com/madskristensen/NpmTaskRunner) for VS 2022 and converted it for NUKE.  It parses the Targets from the `Build.cs` file to populate the Task Nodes and seems to work well.  The code is at [https://github.com/sharpninja/NukeTaskRunner](https://github.com/sharpninja/NukeTaskRunner) and I would like to donate it to the NUKE project.

**Comment by matkoch** [comment]
Isn't this part of the nuke extension already?

**Reply by sharpninja** [reply]
No, it has not been updated to work with VS 2022.

**Comment by sharpninja** [comment]
I have published the extension on Open VSIX Gallery.

https://www.vsixgallery.com/extension/21B985AA-236C-4F50-9704-ABFD443F5814
